test/unit/Cardano/Wallet/CoinSelection/FeeSpec.hs

```diff
@@ -377,15 +408,17 @@ propFeeEstimation (ShowFmt sel, InfiniteList chngAddrs _) =
         (Fee calcFee) = estimateFee cardanoPolicy sel
         (TxSizeLinear (Quantity a) (Quantity b)) = cardanoPolicy
         tx = fromCoinSelection sel
-        size = BL.length $ toLazyByteString $ encodeSignedTx tx
+        encodedTx = toLazyByteString $ encodeSignedTx tx
+        size = BL.length encodedTx
         -- We always go for the higher bound for change address payload's size,
         -- so, we may end up with up to 4 extra bytes per change address in our
         -- estimation.
         margin = 4 * fromIntegral (length $ CS.change sel)
         realFeeSup = ceiling (a + b*(fromIntegral size + margin))
         realFeeInf = ceiling (a + b*(fromIntegral size))
     in
-        property (calcFee >= realFeeInf && calcFee <= realFeeSup)
+        (calcFee >= realFeeInf && calcFee <= realFeeSup, encodedTx)
+            === (True, encodedTx)
```

Comment:
No change in the test logic here, but we'll now display the full transaction binary representation upon failure which is quite handy for debugging!